core/regions/pom.xml

```diff
@@ -94,4 +89,44 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>software.amazon.awssdk</groupId>
+                    <artifactId>codegen-lite-maven-plugin</artifactId>
+                    <version>${awsjavasdk.version}</version>
+                    <executions>
+                        <execution>
+                            <phase>generate-sources</phase>
+                            <goals>
+                                <goal>generate-regions</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
+
+    <profiles>
+        <!-- Build steps executed for services with models -->
```

Comment:
can you explain what this does? not quite getting it.

Reply:
This comment is incorrect as I just copied it from the other but it's just the configuration for running the codegen-lite-maven-plugin. It's exactly copied from the pom.xml of the services module just with the "lite" version.